setup.py

```diff
@@ -20,8 +20,9 @@
 from setuptools import setup, find_packages, Extension
 from setuptools.command.build_ext import build_ext
 
-if sys.version_info[:2] < (3, 5):
-    raise Exception('This version of gensim needs 3.5 or later.')
+PY2 = sys.version_info[:2] == (2, 7)
+if sys.version_info[:2] < (3, 5) and not PY2:
```

Comment:
https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires, please :)

Reply:
Should we do that for 3.8.3, or save it for a non-bugfix release?

Reply:
up to you :)